bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/core/IBigtableTableAdminClient.java

```diff
@@ -128,6 +128,20 @@
    */
   ApiFuture<Void> dropRowRangeAsync(String tableId, String rowKeyPrefix);
 
+  /**
+   * Drops all data in the table.
+   *
+   * @param tableId a {@link String} object.
+   */
+  void dropAllRows(String tableId);
+
+  /**
+   * Asynchronously drops all data in the table
```

Comment:
put a full stop. line 132 and 139. Both sentences are correct grammatically, but technically is it dropping all data or, is it data from the rows of the table?